posthog/models/plugin.py

```diff
@@ -64,3 +67,21 @@ class Meta:
     plugin_config: models.ForeignKey = models.ForeignKey("PluginConfig", on_delete=models.CASCADE)
     key: models.CharField = models.CharField(max_length=200)
     value: models.TextField = models.TextField(blank=True, null=True)
+
+
+class PluginLogEntry(UUIDModel):
+    class Type(models.TextChoices):
+        INFO = "INFO", "info"
+        WARN = "WARN", "warn"
+        ERROR = "ERROR", "error"
```

Comment:
Also `log` and `debug`.

Reply:
Apparently `debug` is only an alias for `log`. https://developer.mozilla.org/pl/docs/Web/API/Console I added `log` only now.

Reply:
Actually I added `debug` too, why not